Frends.CertificateStore/Frends.CertificateStore.cs

```diff
@@ -1,51 +1,179 @@
-﻿using System.ComponentModel;
+﻿using System;
```

Comment:
None of the tasks here would really need to be async as everything is done synchronously.